core-bundle/src/Resources/contao/dca/tl_module.php

```diff
@@ -133,7 +133,8 @@
 		'defineRoot'                  => 'rootPage',
 		'protected'                   => 'groups',
 		'reg_assignDir'               => 'reg_homeDir',
-		'reg_activate'                => 'reg_jumpTo,reg_text'
+		'reg_activate'                => 'reg_jumpTo,reg_text',
+		'formSettings'                => '',
```

Comment:
Judging from the failing functional test these are currently expected to be called `overwriteFormSettings`.

Reply:
Thanks! Fixed in 4487b7e